src/modules/identity-registry/schema.ts

```
@@ -0,0 +1,460 @@
+  update: boolean
+): readonly string[] | RosterFieldUpdate<readonly string[]> | undefined {
+  if (isBlank(value)) return update ? keep : Object.freeze([]);
+  if (value === ROSTER_CLEAR_LITERAL) return update ? clear : Object.freeze([]);
+  if (typeof value !== "string") {
+    errors.push(error(rowNumber, "aliases", "invalid_value"));
+    return undefined;
+  }
+  const aliases: string[] = [];
+  const normalizedSeen = new Set<string>();
+  try {
+    for (const line of value.split(/\r?\n/u)) {
```

ESLint

[error] Move this regular expression to module scope to avoid re-compilation on every call.

(e18e/prefer-static-regex)